spec/factories/piece_factory.rb

```diff
@@ -6,5 +6,6 @@
     in_game true
     association :player
     association :game
+    after(:build) { |piece| piece.class.skip_callback(:create, :after, :populate_board!) }
```

Comment:
You can remove this line. Now that you have the stub in the piece_spec


`allow_any_instance_of(Game).to receive(:populate_board!).and_return true`